src/FixedDataTableRow.js

```diff
@@ -341,7 +373,12 @@ class FixedDataTableRowImpl extends React.Component {
 
     return (
       <div
-        className={joinClasses(className, this.props.className)}
+        className={joinClasses(
+          className,
+          this.props.className,
+          cx('fixedDataTableRowLayout/body'),
+          cx('fixedDataTableRowLayout/rowWrapper')
+        )}
```

Comment:
We have to join the wrapper class with the class of this div .One more div was used for fixedDataTableRowLayout also .We can join that class also to maintain the styling and removing one div also .